modules/manageSectionDisplay.js

```diff
@@ -0,0 +1,23 @@
+export default function showSection(number) {
+  switch (number) {
+    case 0:
+      document.getElementById('book-list-section').style.display = 'flex';
+      document.getElementById('add-book-section').style.display = 'none';
```

Comment:
- [ ] Same here, please use the `ES6` arrow function syntax instead of regular syntax.